west.yml

```diff
@@ -125,6 +133,12 @@ manifest:
       path: modules/alexa-embedded
       revision: face92d8c62184832793f518bb1f19379538c5c1
       remote: alexa
+    - name: cddl-gen
+      remote: oyro
```

Comment:
will there be an official PR when forking/moving this repo into Nordic ?

Especially cause `cddl_gen.py` contains:
```
 Generated with cddl_gen.py (https://github.com/oyvindronningstad/cddl_gen)
```
but that should be updated in that process.
Especially cause the code in that file has the following header:
```
#!/usr/bin/env python3
#
# Copyright (c) 2020 Nordic Semiconductor ASA
#
# SPDX-License-Identifier: Apache-2.0
#
```
and code copyrighted by Nordic should refer to Nordic repos.

@oyvindronningstad why is it `Apacha-2.0` licensed, and not BSD ?

Reply:
This repo will be moved to the nordic organization, and we will have a separate PR for adding it to the west.yml.

Reply:
But adding the repo itself to the `west.yml` does not ensure that the code itself will be reviewed.

Could you somehow make sure that the code itself will be reviewed when moving it ?

Reply:
Well, the code of the cddl_gen repo will end up here in nrfconnect/, so anyone can go and take a look at it, although it's already in use in MCUboot and has thus has seen external review. Since I am still undecided about a name for this repo, let us wait until we have it in this org and we'll take a look together then.

Reply:
>  although it's already in use in MCUboot and has thus has seen external review.

does that mean it has been through a normal review ?
I don't review all code for a new project when it's being included in our manifest, and I don't think i'm special in that sense.
But all files in this repo is copyrighted by Nordic, so maybe we should review our own code with the same eyes as if this code had been added as a series of commits to any other repo.